### PR TITLE
chore: Typescript Cleanup- Part 5 for Errors - urls,cardsViewStoreData,storeActionMessageCreator,axeResultToIssueFilingDataConverter

### PR DESCRIPTION
**Pull request by JeevaniChinthala**
#### Details

As part of Typescript migration added/removed the error causing fields.

##### Motivation

TypeScript 5.0 has marked some options as deprecated. We can temporarily override these deprecations, but the documented plan is for the ability to override the flag to go away in TypeScript 5.5

##### Context

1. **urls: urlsConfiguration**

- **Error:**  Object literal may only specify known properties, and 'urls' does not exist in type 'PropertyIdToConfigurationMap'.
- Added urls property.

2. **cardsViewStoreData={this.props.cardsViewStoreData}**

-  **Error:**  Property 'cardsViewStoreData' does not exist on type 'IntrinsicAttributes & CommonInstancesSectionProps
- Added cardsViewStoreData in CommonInstancesSectionProps added the property in file [common-instances-section-props.ts]

3. **storeActionMessageCreator**

- **Error:** Object literal may only specify known properties, and 'storeActionMessageCreator' does not exist in type 'PageDeps'.
- Removed storeActionMessageCreator as there are no references available for the property.

4. **deps:null**

- **Error:**   Object literal may only specify known properties, and 'deps' does not exist in type 'ReportCollapsibleContainerProps'.
- Getting the error in collapsible-result-section.tsx file.
- Added deps and onExpandToggle properties in ReportCollapsibleContainerProps in file report-collapsible-container.tsx.

5. **selectedIssueFilingServiceData**

- **Error:** Object literal may only specify known properties, but 'selectedIssueFilingServiceData' does not exist in type 'IssueFilingNeedsSettingsContentProps'
- This property is used in props in the file [issue-filing-settings-container.tsx]

6. **isMoveToAssessmentDialogOpen**

- **Error:** Object literal may only specify known properties, but 'isMoveToAssessmentDialogOpen' does not exist in type 'Readonly<DetailsViewCommandBarState>'
- Upon checking previous history and references it is not used anywhere

9. **rel="noopener"**

-  **Error:** Type '{ children: Element[]; action: string; method: string; target: string; rel: string; style: { visibility: "hidden"; }; }' is not assignable to type 'DetailedHTMLProps<FormHTMLAttributes<HTMLFormElement>, HTMLFormElement>'.

10. **value**

- **Error:** Type '{ value: boolean; label: string; onChange: (event: MouseEvent<any, MouseEvent>, checked?: boolean) => void; }' is not assignable to type 'IntrinsicAttributes & ICheckboxProps & { children?: ReactNode; }'.

11. **isLableVisisble**

- **Error:** Property 'isLabelVisible' does not exist on type 'IntrinsicAttributes & IChoiceGroupProps & { children?: ReactNode; }'.

#### Pull request checklist

- [x] Addresses an existing issue: #(https://github.com/microsoft/accessibility-insights-web/pull/6611)
- [ ] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/DetailsView/details-view-initializer.ts [hunk 1]
```diff
@@ -711,8 +706,6 @@ if (tabId != null) {
                 clickHandlerFactory,
                 issuesTableHandler,
                 previewFeatureFlagsHandler,
-                scopingFlagsHandler,
```
[the remark this answers is not in the document]
Comment (JeevaniChinthala) [reply src/DetailsView/details-view-initializer.ts:714]: When 'axeResultToIssueFilingDataConverter' property from the same file is resolved, then we are getting issue for this property checked using tbuild:all. Hence resolved.

File: src/DetailsView/details-view-initializer.ts [hunk 2]
```diff
@@ -711,8 +706,6 @@ if (tabId != null) {
...
-                Assessments,
```
[the remark this answers is not in the document]
Comment (JeevaniChinthala) [reply src/DetailsView/details-view-initializer.ts:715]: When 'axeResultToIssueFilingDataConverter' property from the same file is resolved, then we are getting issue for this property checked using tbuild:all. Hence resolved.

File: src/DetailsView/details-view-initializer.ts [hunk 3]
```diff
@@ -408,7 +406,6 @@ if (tabId != null) {
             const previewFeatureFlagsHandler = new PreviewFeatureFlagsHandler(
                 getAllFeatureFlagDetails(),
             );
-            const scopingFlagsHandler = new PreviewFeatureFlagsHandler(getAllFeatureFlagDetails());
```
[the remark this answers is not in the document]
Comment (JeevaniChinthala) [reply src/DetailsView/details-view-initializer.ts:411]: When 'axeResultToIssueFilingDataConverter' property from the same file is resolved, then we are getting issue for this property checked using tbuild:all. Hence resolved.

File: src/DetailsView/components/tab-stops/tab-stops-instance-section-props-factory.tsx [hunk 4]
```diff
@@ -153,7 +153,6 @@ export const ReportTabStopsInstanceSectionPropsFactory: TabStopsInstanceSectionP
                 content: (
                     <TabStopsReportInstanceList
                         key={`${result.id}-requirement-group`}
-                        requirementId={result.id}
```
[the remark this answers is not in the document]
Comment (JeevaniChinthala) [reply src/DetailsView/components/tab-stops/tab-stops-instance-section-props-factory.tsx:156]: Yes. Tested

**Review by madalynrose — CHANGES_REQUESTED** [review]

File: src/DetailsView/components/save-assessment-button.tsx [hunk 5]
```diff
@@ -74,9 +74,6 @@ export const SaveAssessmentButton = NamedFC<SaveAssessmentButtonProps>(
                         >
                             <Stack.Item grow disableShrink>
                                 <Checkbox
-                                    value={
```
[the remark this answers is not in the document]
Comment (JeevaniChinthala) [reply src/DetailsView/components/save-assessment-button.tsx:77]: Updated

File: src/common/components/visualization-toggle.tsx [hunk 6]
```diff
@@ -30,7 +30,7 @@ export class VisualizationToggle extends React.Component<VisualizationToggleProp
             componentRef: this.props.componentRef,
             onBlur: this.props.onBlur,
             onFocus: this.props.onFocus,
-            'data-automation-id': this.props['data-automation-id'],
+            ...{ 'data-automation-id': this.props['data-automation-id'] },
         };
```
[the remark this answers is not in the document]
Comment (JeevaniChinthala) [reply src/common/components/visualization-toggle.tsx]: Updated.

File: src/reports/components/report-sections/report-collapsible-container.tsx [hunk 7]
```diff
@@ -14,6 +14,7 @@ export interface ReportCollapsibleContainerProps {
     containerClassName?: string;
     buttonAriaLabel?: string;
     testKey?: string;
+    onExpandToggle: (event: React.MouseEvent<HTMLDivElement>) => void;
```
[the remark this answers is not in the document]
Comment (JeevaniChinthala) [reply src/reports/components/report-sections/report-collapsible-container.tsx]: Updated.

**Review by madalynrose — APPROVED** [review]

**Pull request merged**